pkg/status/dist/templates/jmxfetch.tmpl

```diff
@@ -1,32 +1,37 @@
 ========
 JMXFetch
 ========
-{{- with .JMXStatus -}}
-{{- if and (not .timestamp) (not .checks)}}
-no JMX status available
-{{- else }}
-    initialized checks:
-{{- range $check,$instances := .checks.initialized_checks }}
-        {{ $check -}}
-{{- range $instance := $instances }}
-{{- range $k,$v := $instance }}
-            {{ $k }} : {{ $v }}
-{{- end -}}
-{{- end -}}
+{{ with .JMXStatus }}
+  {{- if and (not .timestamp) (not .checks) }}
+  no JMX status available
+  {{- else }}
+  Initialized checks:
+  ===================
```

Comment:
Smallest nit ever, do you think you could take off the ":" here? For consistency 😁